### PR TITLE
fix(Swift): Addressing some gaps in Geo Gen2 doc pages

### PR DESCRIPTION
**Pull request by ruisebas**
#### Description of changes:

This PR addresses the following issues within the Swift Gen2 documentation for Geo:

##### Set up Amplify Geo
- Removed unnecessary block for installation instructions, and rewrote them to be consistent with other platforms
- Added explicit instruction to generate and add the `amplify_outputs.json` file to the project.
- Fixed some outdated signatures being used
- Added missing import statement to the initialization code 

##### Work with maps
- Removed outdated UIKit code mentioning both "Stable release" (i.e. v1) and "Developer Preview" (i.e. v2) 

##### Configure location search
- Added missing `aws_region` to the backend code snippet.

##### Work with location search
- Fixed indentation in a code snippet

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [X] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [X] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by hdworld11 — APPROVED** [review]

**Review by hdworld11 — DISMISSED** [review]

File: src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx [hunk 1]
```diff
@@ -209,21 +209,18 @@ dependencies {
...
+  <Callout>
 
-1. Choose the dependency rule **Up to Next Major Version**, as it will use the latest compatible version of the dependency, then click **Add Package**.
+  **Note:** **Up to Next Major Version** should be selected from the **Dependency Rule** dropdown.
 
-1. Lastly, choose **AWSLocationGeoPlugin**, **AWSCognitoAuthPlugin**, and **Amplify**. Then click Finish.
+  </Callout>
 
-</Block>
+1. Lastly, add **AWSLocationGeoPlugin**, **AWSCognitoAuthPlugin**, and **Amplify** to your target. Then click **Add Package**.
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx:222]:
```suggestion
3. Lastly, add **AWSLocationGeoPlugin**, **AWSCognitoAuthPlugin**, and **Amplify** to your target. Then click **Add Package**.
```

File: src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx [hunk 2]
```diff
@@ -340,12 +337,23 @@ Initialized Amplify
 </InlineFilter>
 
 <InlineFilter filters={['swift']}>
-To initialize the Amplify Geo, use the `Amplify.addPlugin()` method to add the AWSLocationGeoPlugin. Next, finish configuring Amplify by calling `Amplify.configure()`.
+<Callout warning>
+Make sure to generate the `amplify_outputs.json` file by running the following command: 
+
+```bash title="Terminal" showLineNumbers={false}
+npx ampx sandbox
+```
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx:345]: should this be moved outside the swift block? I think we should keep the prose relatively the same across platforms (with a few exceptions) for maintainability and a concise UX

Comment (ruisebas) [reply src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx:345]: We could explore doing that in the future as a docs clean up, but right now each platform is doing its own thing

**Review by hdworld11 — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx (hunk 2 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx:345]: platform-specific callouts were (mostly) cleaned up when content was ported into this new structure. swift has the callout to move the file into the project, which is fine, but prose should be relatively the same

**Pull request merged**